app/javascript/mastodon/features/compose/util/language_detection_with_laude.js

```diff
@@ -0,0 +1,38 @@
+import lande from 'lande';
```

Comment:
The file is called `language_detection_with_laude.js` but the library is called `lande`.